docs/cost/ids/core.md

```diff
@@ -0,0 +1,267 @@
+# Core standard
+
+<style>
+.wy-nav-content {
+  max-width: 1200px;
+}
+</style>
+
+## Project level
+
+## Identification
```

Comment:
The heading levels below level 2 don't seem correct.

Reply:
Fixed in https://github.com/open-contracting/infrastructure/pull/422/commits/c6104c6125a01841151d4f10f2d875b438014aac